CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+# dbt_zendesk v0.14.0
+
+[PR #136](https://github.com/fivetran/dbt_zendesk/pull/136) includes the following changes:
+
+## Bug Fixes
+- Converted the `zendesk__sla_policies` metric for `sla_elapsed_time` to be reported in minutes to the second as opposed to just the nearest rounded minute. This ensures more accurate reporting.
+- Updated the `int_zendesk__reply_time_combined` model to additionally account for the following business hour scenarios as they were erroneously being filtered out in previous versions of the package:
+   - A ticket is first replied to outside SLA schedules
+   - A ticket is has not yet received an agent reply
+- Included additional logic within the `int_zendesk__ticket_schedules` model to more accurately select the **active** default schedule.
+  - Previously the model could possibly select a deleted schedule. This update ensures only an active schedule is selected.
+- Overhauled the logic used to calculate `sla_breach_at` within the `zendesk__sla_policies` and upstream models for reply time SLAs. It was found this field was inconsistent with the actual breach/achieve time of an SLA. The overhaul should now ensure reply time SLA is accurate to either be the time of the SLA breach or achieve event.
```

Comment:
Perhaps a subbullet with more detail on how the logic was overhauled? I feel like it'll be good for recordkeeping since this portion was thicc, and it would add color in the case we ever need to return to this.

Reply:
Oh after reviewing `zendesk__sla_policies` I see you added a lot of detailed comments describing the logic! That should suffice too

Reply:
Good callout. I added some more comments in the intermediate model to allow us to better understand this in future code updates and also added a sub-bullet to highlight the specific adjustment.

Reply:
Added.